portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -60,16 +60,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         // Create ArrayList of comments and populate from the database
         ArrayList<Comment> commentsList = new ArrayList<Comment>();
         Iterator resultIterator = results.asIterable().iterator();
-        for (int i = 0; i < Integer.parseInt(request.getParameter(NUM_COMMENTS_PARAMETER)); i++) {
-            if(resultIterator.hasNext()) {
-                Entity commentEntity = (Entity) resultIterator.next();
-                String text = (String) commentEntity.getProperty(TEXT_PROPERTY);
-                long timestamp = (long) commentEntity.getProperty(TIMESTAMP_PROPERTY);
-                    
-                Comment comment = new Comment(text, timestamp);
-                commentsList.add(comment);
-                System.out.println("Item fetched" + System.currentTimeMillis());
+        int numComments = Integer.parseInt(request.getParameter(NUM_COMMENTS_PARAMETER));
```

Comment:
I think the error cases are still not handled:
1. What happens when `request.getParameter(NUM_COMMENTS_PARAMETER)` is not a number?
2. What happens when `request.getParameter(NUM_COMMENTS_PARAMETER)` is not provided?